apps.json

```diff
@@ -1556,6 +1556,16 @@
         ],
         "url": "https://github.com/YunoHost-Apps/kodi_ynh"
     },
+    "komga": {
+        "branch": "master",
+        "category": "reading",
+        "revision": "HEAD",
+        "state": "working",
+        "subtags": [
+            "comics"
+        ],
```

Comment:
That subtag doesn't exists though, so proposing to drop it (or maybe "books" but idk)

Reply:
hmm, sure books is alright for now

Reply:
changed it